src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java

```diff
@@ -148,11 +140,10 @@ public User getUserById(int id) {
 
     @Override
     public boolean updateUser() {
-        String sql = "UPDATE login SET username = ?, password = ?, entity_id = ?, date_modified = ? WHERE id = ?";
         User user = new User();
 
         try (Connection con = ConnectionHelper.getConnection();
-             PreparedStatement stmt = con.prepareStatement(sql)) {
+             PreparedStatement stmt = con.prepareStatement(UPDATE_USER_STATEMENT)) {
```

Comment:
fix the try-with-resource. remove preparedstatement from try()

Reply:
okay napo sir